app/config/config_dev.yml

```diff
@@ -14,12 +14,20 @@ web_profiler:
 monolog:
     handlers:
         main:
-            type:   stream
-            path:   "%env(SYMFONY_LOG)%"
-            level:  info
+            type: stream
+            path: "%env(SYMFONY_LOG)%"
```

Comment:
@voronkovich In any case it would be `APP_LOG_PATH` (as it's the path to the log file), but I'm no sure why is it configurable here, so by default it's `'%kernel.logs_dir%/%kernel.environment%.log'` :/